tests/azure-sdk-for-cpp/Makefile

```diff
@@ -48,4 +55,4 @@ clean:
 	rm -rf $(APPDIR) $(ROOTFS)
 
 get-executables:
-	find $(RUNTEST) $(MYST_EXEC) $(OPTS) $(ROOTFS) /azure-sdk-for-cpp/build/sdk -type f -executable -print | grep test
+	find $(MYST_EXEC) $(OPTS) $(ROOTFS) /azure-sdk-for-cpp/build/sdk -type f -executable -print | grep test
```

Comment:
Seems this target is not used anymore? What's the purpose of it?

Reply:
I wondered this myself. There's no mention of it but I guess it is kept around as a convenience for finding test executables

Reply:
This target is used when we extract the binaries from the docker container, mostly it is kept for convenience

Reply:
We can find the tests from appdir without invoking mystikos. In addition, I suspect the command should be:
```
$(MYST_EXEC) $(OPTS) $(ROOTFS) find /azure-sdk-for-cpp/build/sdk -type f -executable -print |  grep test
```

Reply:
Good catch Xuejun